daemon/src/taker_cfd.rs

```diff
@@ -73,24 +70,44 @@ impl Actor {
     async fn handle_latest_offers(&mut self, msg: offer::taker::LatestOffers) {
         self.offers.insert(msg.0.clone());
 
+        for i in msg.0.iter() {
+            tracing::info!(
+                "Received new offer: {:?} with price {}",
+                i.position_maker.counter_position(),
+                i.price
+            );
+            self.latest_offers.insert(
+                (i.contract_symbol, i.position_maker.counter_position()),
+                i.clone(),
+            );
+        }
+
         if let Err(e) = self.projection_actor.send(projection::Update(msg.0)).await {
             tracing::warn!("Failed to send current offers to projection actor: {e:#}");
         };
     }
 
     async fn handle_propose_settlement(&mut self, msg: ProposeSettlement) -> Result<()> {
-        let ProposeSettlement {
-            order_id,
-            bid,
-            ask,
-            quote_timestamp,
-        } = msg;
+        let ProposeSettlement { order_id } = msg;
 
         let cfd = self.db.load_open_cfd::<Cfd>(order_id, ()).await?;
 
-        let proposal_closing_price = market_closing_price(bid, ask, Role::Taker, cfd.position());
```

Comment:
🛠️ We should remove the function `market_closing_price` and also use the maker's price in the projection - otherwise what we show in the UI does not match what is actually used!